pkg/component/extensions/operatingsystemconfig/original/components/rootcertificates/component.go

```diff
@@ -118,7 +117,6 @@ ConditionPathExists=!` + kubelet.PathKubeconfigReal + `
 [Service]
 Type=oneshot
 ExecStart=` + pathUpdateLocalCaCertificates + `
-ExecStartPost=/bin/systemctl restart ` + docker.UnitName + `
```

Comment:
Could this potentially lead to issues when pulling the hyperkube images via docker?

Reply:
What do you have in mind? `hyperkube` is pulled before the `rootcertificates` component is deployed, so if this process needed a custom CA certificate, this would have failed already today (respectively, the custom CA certificate must get provisioned via different means already today).

Reply:
You can ignore this comment.
I wasn't sure about the order of operations so I wanted to double check. I was mainly confused because of the checks [here](https://github.com/gardener/gardener/blob/0edfd8e6b69b33f81f9c88e52d0b9e7a50ce1403/pkg/component/extensions/operatingsystemconfig/original/components/rootcertificates/component.go#L117) and [here](https://github.com/gardener/gardener/blob/0edfd8e6b69b33f81f9c88e52d0b9e7a50ce1403/pkg/component/extensions/operatingsystemconfig/original/components/rootcertificates/component.go#L114), but I missed the fact that the units as a whole are enabled after the hyperkube image is downloaded.